models/base/shopify_order_items_tax.sql

```diff
@@ -0,0 +1,10 @@
+select
+
+	_sdc_level_0_id,
```

Comment:
Can we rename these fields to be less stitch-specific? It think we can alias `_sdc_level_0_id` to `order_id` and `_sdc_level_1_id` to `order_item_id`. Same for `_sdc_level_0_id` in `shopify_order_items.sql`

We do something similar here: https://github.com/fishtown-analytics/shopify/blob/master/models/base/shopify_order_items.sql#L5